README.md

```diff
@@ -1,87 +1,161 @@
 
-## Writing a tutorial
+Inside, you will find:
+- `/content/` - Directory containing the tutorial markdown files
+- `/content/index.mdx` - The main markdown file for your tutorial
+- `/content/assets/` - Place any images used in your tutorial here, linking to images is explained in the tutorial template
+- `/builderdao.config.json` - Tutorial metadata
+- `/builderdao.lock.json` - Configuration lockfile
 
-By running `yarn install` in the root of the project you should have a BuilderDAO CLI available
+## Commit, Push, Create a PR
 
-### Initializing tutorial
+> If this is your first time pushing a commit to a tutorial branch, you will need to set the upstream and `git` will prompt you to do so, otherwise you can just `git push`.
 
-In order to initialize tutorial run
+Once you are finished writing the first draft of your tutorial, save your changes and then commit and push to your fork. Optionally, you can check the files being staged with `git status`: 
 
-```bash
-builderdao tutorial init
+```text
+git status
+git add .
+git commit -m "<YOUR_TUTORIAL_TITLE>"
+git push
 ```
 
-You will be prompted with few questions about your tutorial. After you are done you will have a boilerplate to start writing your tutorial.
-A new folder will be created for your tutorial. Name of the folder is the same as the tutorial slug.
-Inside of this folder you will find:
-- `content` - folder with tutorial markdown files
-- builderdao.config.json - file with tutorial metadata
-- builderdao.lock.json - file with tutorial configuration. THIS FILE SHOULD NOT BE MODIFIED MANUALLY
+> Remember to replace `<YOUR_TUTORIAL_TITLE>` and `<SLUG>` with the correct values for your tutorial. For example, if the title was "HTML 101", the slug would be "html-101".
+
+Now go to your fork of the repo on GitHub (`https://github.com/<YOUR_GITHUB_USERNAME>/kafe`) and [create a Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request). Make sure that the _base branch_ of your PR is the `dev` branch of the Kafè repo.
+
+## Publishing
 
-### Publishing
+Once your PR is active, the assigned reviewers will look it over carefully and make suggestions for improvement, point out errors and typos, and work with you to ensure a high quality tutorial.
 
-Once you are done writing your tutorial, commit your changes, push your tutorial branch to GitHub and create a PR.
-After 2 assigned reviewers approve your tutorial and merge it in, your tutorial will be automatically published.
+After both reviewers have approved your tutorial, it will be merged. Your tutorial will be automatically published after it is merged with no further action required. It will then be added to the index and become visible on Kafé at `https://dev.builderdao.io/learn/<SLUG>`. Congratulations! 🎉
```

Comment:
I'd kill "congratulations" here, and change the last few sentences to this:
"After both reviewers have approved your guide, it will be merged and automatically published. Kafé will issue the publishing rewards directly to your wallet, and the guide will become available on the Kafé platform.

Your guide is now ready to share with the world!"


Can we also add a section titled "Contributing"? With something like, 
"If you're interesting in contributing, we'd love to have you! Please take a look at the contribution guidelines and reach out." ---> we need to write some simple contribution guidelines. I'll create a ticket with an outline and request feedback from everyone